control_software/scripts/hera_feng_start.sh

```diff
@@ -7,11 +7,10 @@ set -e
 
 echo hera_feng_start.sh: Initializing F-Engines > $LOGFILE
 date >> $LOGFILE
-hera_snap_feng_init.py -p -i -d &>> $LOGFILE
+hera_snap_feng_init.py -p -i -d -s -e &>> $LOGFILE
```

Comment:
Startup is working, why change now?

Reply:
Because a) this simplifies the number of scripts and steps, and b) I think it is actually a liability to rebuild the list of hosts between these two stages.